docs/widgets/input.md

```diff
@@ -26,7 +26,8 @@ The example below shows how you might create a simple form using two `Input` wid
 
 You can supply one or more *[validators][textual.validation.Validator]* to the `Input` widget to validate the value.
 
-When the value changes or the `Input` is submitted, all the supplied validators will run.
+All the supplied validators will run when the value changes, the `Input` is submitted, or focus moves _out_ of the `Input`.
+This can be customized via the attribute [`prevent_validation_on`][textual.widgets.Input.prevent_validation_on].
```

Comment:
We need to inform the reader what the three possible values are, and what they mean. Also, this is referring to the old attribute name.